packages/broker/src/plugins/operator/inspectSuspectNode.ts

```diff
@@ -1,16 +1,36 @@
-import { Logger } from '@streamr/utils'
+import { EthereumAddress, Logger } from '@streamr/utils'
 import { VoteOnSuspectNodeHelper } from './VoteOnSuspectNodeHelper'
+import { inspectTarget } from './inspectionUtils'
+import { toStreamPartID } from '@streamr/protocol'
+import { StreamrClient } from 'streamr-client'
+import { fetchRedundancyFactor as _fetchRedundancyFactor } from './fetchRedundancyFactor'
 
 const logger = new Logger(module)
 
 export async function inspectSuspectNode(
-    sponsorship: string,
-    targetOperator: string,
+    sponsorship: EthereumAddress,
+    targetOperator: EthereumAddress,
     partition: number,
-    voteOnSuspectNodeHelper: VoteOnSuspectNodeHelper
+    voteOnSuspectNodeHelper: VoteOnSuspectNodeHelper,
+    streamrClient: StreamrClient,
+    abortSignal: AbortSignal,
+    fetchRedundancyFactor = _fetchRedundancyFactor,
+    heartbeatLastResortTimeoutInMs = 60 * 1000,
```

Comment:
What does the "last resort" part of the parameter name reflect?

Reply:
Rename to `heartbeatTimeoutInMs`